sdk/digitaltwins/digital-twins-core/tests.yml

```diff
@@ -7,6 +7,7 @@ stages:
       ServiceDirectory: digitaltwins
       MatrixFilters:
         - TestType=^(?!sample).*
```

Comment:
This is an old way of configuring min max testing that won't work anymore. Min/max is enabled by default now in the platform matrix: https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/stages/platform-matrix.json#L58

Reply:
Would reverting back to what it was by removing the `TestMinMax: true` line be sufficient here? Or does something else need to be done?

Reply:
Yep, just undo the change and it should work as intended. Let me know if you see different behavior than expected. You should see two test jobs like:

![image](https://user-images.githubusercontent.com/1020379/123719134-c5a67000-d84e-11eb-9487-bef05bb48686.png)

Reply:
Unfortunately, I don't see those jobs after removing the line `tests.yml`. @benbp

Reply:
You may have been looking at the CI tests, not the live tests? I kicked off a live test run, which should have them.

Reply:
Yep, that's what it was. Thanks!